lib/packets/resultset_header.js

```diff
@@ -72,6 +73,10 @@ class ResultSetHeader {
             stateChanges.trackStateChange = packet.readLengthCodedString(
               encoding
             );
+          } else if (type === sessionInfoTypes.STATE_GTIDS) {
+            const something = packet.readLengthCodedString(encoding);
```

Comment:
maybe better to be a bit more verbose:

```suggestion
            // TODO: find if the first length coded string means anything. Usually comes as empty
            const _unknownString = packet.readLengthCodedString(encoding);
```

Reply:
Thanks, I updated the code and also add the skip eslint check for this line.